java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1364,4 +1364,55 @@ public void zrange_with_different_types_of_keys(BaseClient client) {
                 assertThrows(ExecutionException.class, () -> client.zrangeWithScores(key, query).get());
         assertTrue(executionException.getCause() instanceof RequestException);
     }
+
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void zcount(BaseClient client) {
+        String key1 = UUID.randomUUID().toString();
+        String key2 = UUID.randomUUID().toString();
+        Map<String, Double> membersScores = Map.of("one", 1.0, "two", 2.0, "three", 3.0);
+        assertEquals(3, client.zadd(key1, membersScores).get());
+
+        assertEquals(
+                3,
+                client
+                        .zcount(key1, InfScoreBound.NEGATIVE_INFINITY, InfScoreBound.POSITIVE_INFINITY)
```

Comment:
Update imports
```suggestion
                        .zcount(key1, NEGATIVE_INFINITY, POSITIVE_INFINITY)
```
or define constants
```suggestion
                        .zcount(key1, pInf, nInf)
```

Reply:
I can't do the first one as there are 2 kinds of infinities one is InfScoreBount and the other is InfLexBound. If we decide on something I will have to update all commands that actually these values that are already merged.

Reply:
Oh I see
You can define local constants to avoid such code formatting if you want